.github/workflows/codespell.yml

```
@@ -18,4 +18,4 @@ jobs:
         with:
           ignore_words_list: crate,raison
           exclude_file: .gitignore
-          skip: package-lock.json
+          skip: package-lock.json,./src/generators/mandoc/template.1
```

```suggestion
          skip: package-lock.json, ./src/generators/mandoc/template.1
```
does adding a space work? it would be a hair easier to read